src/components/home/lists/List.js

```diff
@@ -44,11 +44,13 @@ export default function List({ list }) {
     };
     updateExistingItem(updatedItem);
 
-    // setItems((oldItems) => {
-    //   return oldItems.map((oldItem) =>
-    //     oldItem.id == item.id ? updatedItem : oldItem
-    //   );
-    // });
+    updatedItem._version += 1;
+
+    setItems((oldItems) => {
+      return oldItems.map((oldItem) =>
+        oldItem.id == item.id ? updatedItem : oldItem
+      );
+    });
```

Comment:
While this does make the change occur immediately in the UI, I found while testing that after checking/unchecking a single item, updating items will break entirely for that list (idk why). My previous approach was to just let the subscription update it, but maybe you can fix the bug?

Reply:
Will check this out.
Without this addition, checking itmes was not working dor me